src/Avatar/Avatar.js

```diff
@@ -68,18 +77,23 @@ Identifier.propTypes = {
     glyph: PropTypes.string,
     /** Localized text for label */
     label: PropTypes.string,
-    /** Sets a style variation for a modified appearance: 'circle', 'transparent' */
-    modifier: PropTypes.oneOf(IDENTIFIER_MODIFIERS),
-    /** Applies an aria-role. Set to button if Identifier opens a Popover or Dialog */
+    /** Apply placeholder background color */
+    placeholder: PropTypes.bool,
+    /** Applies an aria-role. Set to button if Avatar opens a Popover or Dialog */
     role: PropTypes.string,
-    /** Size of the component:  'xxs',
+    /** Size of the component:
     'xs',
     's',
     'm',
     'l',
-    'xl',
-    'xxl' */
-    size: PropTypes.oneOf(IDENTIFIER_SIZES)
+    'xl' */
+    size: PropTypes.oneOf(AVATAR_SIZES),
+    /** Apply tile background color */
+    tile: PropTypes.bool,
+    /** Use transparent background */
+    transparent: PropTypes.bool,
+    /** Include a zoom icon */
```

Comment:
Do you mind changing these new boolean's description to "Set to **true** to...." to follow the pattern in other components. I've been trying to think of a better way to get consistency in these so I'm all ears if anyone thinks of a better solution!